src/char/int_rodex.c

```diff
@@ -216,7 +216,7 @@ static int inter_rodex_fromsql(int char_id, int account_id, int8 opentype, int64
 #else
 			msg.opentype = opentype;
 #endif
-#if PACKETVER < 20160601
+#if PACKETVER < 20180601
```

Comment:
I believe this is not an intended changes.
try not to search and replace using the sole keyword `2016`.

Reply:
wait pls

Reply:
Fixed : 5369c79